.github/action/git-version/action.yaml

```
@@ -6,6 +6,12 @@ inputs:
     description: Path to the git-version config file
     required: true
 
+outputs:
+  fullSemVer:
+    description: The full semantic version
+  preReleaseTag:
+    description: The pre-release tag
+
```

Tu peux ptetre plutot faire ca ?

```suggestion
outputs:
  fullSemVer:
    description: The full semantic version
    value: ${{ steps.gitversion.outputs.fullSemVer }}
  preReleaseTag:
    description: The pre-release tag
    value: ${{ steps.gitversion.outputs.preReleaseTag }}

```

---

Ca marche ca ? j'ai hésité du coup j'ai fait confiance à copilot

---

car du coup avec ca je peux éviter la step d'export à la fin non ?

---

vue ensemble c'est done.